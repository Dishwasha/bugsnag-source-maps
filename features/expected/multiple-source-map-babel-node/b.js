"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = void 0;

class B {
  hey() {
    console.log('hey');
  }

}

exports.default = B;
//# sourceMappingURL=b.js.map
