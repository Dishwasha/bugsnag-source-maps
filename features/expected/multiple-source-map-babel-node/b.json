{
    "version": 3,
    "sources": [
        "src/lib/b.js"
    ],
    "names": [
        "B",
        "hey",
        "console",
        "log"
    ],
    "mappings": ";;;;;;;AAAe,MAAMA,CAAN,CAAQ;AACrBC,EAAAA,GAAG,GAAG;AACJC,IAAAA,OAAO,CAACC,GAAR,CAAY,KAAZ;AACD;;AAHoB",
    "sourcesContent": [
        "export default class B {\n  hey() {\n    console.log('hey')\n  }\n}\n"
    ],
    "file": "b.js"
}
